tests/runner.test.ts

```diff
@@ -181,6 +194,31 @@ describe('Runner', () => {
         new Error(`Invalid fiddle: "'invalid-fiddle'"`),
       );
     });
+
+    it('spawns a subprocess with ASAR path when runFromAsar is true', async () => {
+      const runner = await createFakeRunner({});
+      (child_process.spawn as jest.Mock).mockReturnValueOnce(mockSubprocess);
+
+      await runner.spawn('12.0.1', '642fa8daaebea6044c9079e3f8a46390', {
+        out: {
+          write: mockStdout,
+        } as Pick<Writable, 'write'> as Writable,
+        runFromAsar: true,
+      });
+
+      expect(child_process.spawn).toHaveBeenCalledTimes(1);
+      expect(child_process.spawn).toHaveBeenCalledWith(
+        '/path/to/electron/executable',
+        ['/path/to/fiddle/app.asar'],
+        {
+          args: [],
+          headless: false,
+          out: expect.any(Object) as Writable,
+          showConfig: true,
+          runFromAsar: true,
+        },
```

Comment:
```suggestion
        expect.anything(),
```

We don't care about these arguments, so allow them to be anything.